site/viamguides/src/qrcode/qrcode.md

```diff
@@ -0,0 +1,131 @@
+  - 1 - microSD card to use with your Pi
+  - 1 - power supply for your Pi
+  - 1 - USB web camera
+
+### What You’ll Need
+
+- All the hardware components listed in prerequisites.
+- Sign up for a free Viam account, and then [sign in](https://app.viam.com/fleet/dashboard) to the Viam app
+
+### What You’ll Learn
+
+- How to use a webcam to scan and detect QR codes
+- How to configure and test a device's components using Viam
+- How to use a vision service in Viam
+
+### Watch the Video
+
+Follow along with the step-by-step video.
+
+<!-- ------------------------ -->
+
+## Configure your machine and peripherals
+
+Duration: 3
+
+### Configure your machine
+
+1. In [the Viam app](https://app.viam.com/fleet/dashboard) under the **LOCATIONS** tab, create a machine by typing in a name and clicking **Add machine**.
+   ![add machine](assets/addMachine.png)
+1. Click **View setup instructions**.
+   ![setup instructions](assets/awaiting.png)
+1. To install `viam-server` on the Raspberry Pi device that you want to use to communicate with and control your webcam, select the `Linux / Aarch64` platform for the Raspberry Pi, and leave your installation method as [`viam-agent`](https://docs.viam.com/how-tos/provision-setup/#install-viam-agent).
+   ![select platform](assets/platform.png)
+1. Use the `viam-agent` to download and install `viam-server` on your Raspberry Pi. Follow the instructions to run the command provided in the setup instructions from the SSH prompt of your Raspberry Pi.
+   ![installation agent](assets/ssh.png)
+1. The setup page will indicate when the machine is successfully connected.
+   ![successful toast](assets/success.png)
+
+### Add your USB webcam
+
+1. Connect the USB webcam to the Raspberry Pi.
+1. In [the Viam app](https://app.viam.com/robots), find the **CONFIGURE** tab. It's time to configure your hardware.
+1. Click the **+** icon in the left-hand menu and select **Component**.
+   ![select component](assets/component.png)
+1. Select `camera`, and find the `webcam` module. This adds the module for working with a USB webcam. Leave the default name `camera-1` for now.
+   ![select webcam](assets/camera.png)
+1. Notice adding this component adds the webcam hardware component called `camera-1`. The collapsible panel on the right corresponds to the part listed in the left sidebar. From the **Attributes** section of the panel, select a `video_path`.
+   ![select video path](assets/videoPath.png)
+1. Click **Save** in the top right to save and apply your configuration changes.
+1. At the bottom of the `camera-1` panel, expand the **TEST** section to ensure you have configured the camera properly.
+   ![test camera](assets/test.png)
+   > aside negative
+   > If any problems occur, check under the **LOGS** tab to see what might be going wrong.
```

Comment:
Should we link to the "common errors" section of the webcam component docs for further help if they need it? https://docs.viam.com/components/camera/webcam/#troubleshooting

```suggestion
   > If any problems occur, check under the **LOGS** tab to see what might be going wrong. [Refer to the troubleshoot guide if needed.](https://docs.viam.com/components/camera/webcam/#troubleshooting)
```